test/test_sync.cpp

```diff
@@ -190,10 +190,11 @@ TEST(SyncTimePoint, RelOperators)
     EXPECT_FALSE(a < b);
 }
 
+#ifndef ENABLE_STDCXX_SYNC
 TEST(SyncTimePoint, OperatorMinus)
 {
     const int64_t                  delta = 1024;
-    const steady_clock::time_point a(numeric_limits<uint64_t>::max());
+    const steady_clock::time_point a(steady_clock::time_point::max());
```

Comment:
Why here and not in the line below? At least explain in the comment.

Reply:
Reverted the change